service/src/main/java/uk/gov/hmcts/reform/fpl/model/RespondentSolicitor.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+import uk.gov.hmcts.reform.ccd.model.Organisation;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+@Data
+@Builder(toBuilder = true)
+@Jacksonized
+@Valid
+public class RespondentSolicitor {
+    private String firstName;
+    private String lastName;
+    @NotNull(message = "Enter the respondent's solicitor's email address")
+    private String email;
```

Comment:
is this working? i seemed to be able to continue through the respondent event without email address